infrastructure/src/main/java/org/corfudb/infrastructure/ServerHandshakeHandler.java

```diff
@@ -117,22 +115,15 @@ public void channelRead(ChannelHandlerContext ctx, Object m) throws Exception {
 
         // Store clientID as a channel attribute.
         ctx.channel().attr(clientIdAttrKey).set(clientId);
-        log.info("channelRead: Handshake validated by Server.");
-        log.debug("channelRead: Sending handshake response: Node Id: {} Corfu Version: {}",
```

Comment:
Should we keep the debug log?

Reply:
+1 this Corfu Version allows to catch when there is a Corfu mismatch

Reply:
"If there is a mismatch let's add more check to prevent inconsistency rather than printing an endless number of messages" - i can't agree more with you Slava but at the same time we cannot lose debug ability of the product without a work around.  Please do not merge this change without the work around that you have suggested.